### PR TITLE
Adds documentation for labels flag

### PR DESCRIPTION
**Pull request by trelore**

### CODE REVIEW

**Review by ajp-io — APPROVED** [review]

File: src/markdown-pages/install-with-kurl/advanced-options.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The install scripts are idempotent. Re-run the scripts with different flags to c
 | ignore-remote-upgrade-prompt     | Bypass prompt to upgrade remotes. This is useful for automating upgrades.                                         |
 | installer-spec-file              | This flag takes the path to a ‘patch’ yaml file. The config in this patch will be merged with the existing installer yaml, taking precedence where there is conflict, and will change the installation based on the final config. |
 | kurl-install-directory           | Override the base directory where kURL will install its dependencies. The path will be suffixed with "/kurl/" ("{kurl-install-directory}/kurl/"). This directory must be writeable by the kURL installer and must have sufficient disk space (5 GB). (default "/var/lib/") |
+| labels                           | Apply the given labels to the node. For example: \"labels=gpu=enabled,type=data\".                                |
```
Comment (JustinN-Replicated) [inline src/markdown-pages/install-with-kurl/advanced-options.md:26]: We may want to adjust slightly
> Apply the given labels to the added node(s). For example: \"labels=gpu=enabled,type=data\".

Comment (ajp-io) [reply src/markdown-pages/install-with-kurl/advanced-options.md:26]: Already made a change almost identical to that, so I'll resolve this.

Comment (jdewinne) [reply src/markdown-pages/install-with-kurl/advanced-options.md:26]: So sad the doc didn't pick up on my example with `appels`, `banana`, `orange`, ....
https://github.com/replicatedhq/kURL/pull/2445#issue-1073839160
🤣

**Review by paigecalvert — APPROVED** [review]

File: src/markdown-pages/install-with-kurl/advanced-options.md (hunk 1 above)
Comment (ajp-io) [reply src/markdown-pages/install-with-kurl/advanced-options.md:26]: Shoot, I missed that!

**Pull request merged**